src/content/configuration/module.mdx

```diff
@@ -275,19 +281,39 @@ module.exports = {
 }
 ```
 
-## `dataUrlCondition.minSize`
+When a function given, returning `true` tells webpack to inject the module into the bundle as Base64-encoded string, otherwise module file will be emitted into output directory.
```

Comment:
```suggestion
When a function given, returning `true` tells webpack to inject the module into the bundle as Base64-encoded string, otherwise module file will be emitted into the output directory.
```